048-IncrementalSynapsePipelines/Coach/README.md

```diff
@@ -0,0 +1,34 @@
+# What The Hack - Incremental Synapse Pipelines <br> <i>Utilizing Change Data Capture in Azure SQL</i> <br> Coach's Guide 
+
+## Introduction
+Welcome to the coach's guide for the Automating Incremental Pipelines What The Hack. Here you will find links to specific guidance for coaches for each of the challenges.
+
+Also remember that this hack includes a optional [lecture presentation](Automating-Incremental-Pipelines.pptx?raw=true) that features key topics for this What the Hack. 
+
+## Solutions
+- Challenge 0: **[Setup the Source and Target Environments](Solution-00.md)**
```

Comment:
This is a silly title name. Can you do better?

Reply:
No I cannot.

Reply:
Fine.